tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -72,15 +71,13 @@ def test_nodereplacement_proactive(self, pvc_factory, pod_factory, dc_pod_factor
                     f"results of this test run are all invalid.")
 
         elif config.ENV_DATA['platform'].lower() == constants.VSPHERE_PLATFORM:
-            pytest.skip("Skipping add node in Vmware platform due to "
-                        "https://bugzilla.redhat.com/show_bug.cgi?id=1844521"
-                        )
```

Comment:
we need this bz decorator if ocp version is greater then 4.5

Reply:
What about this issue which @prsurve pointed out?

Reply:
Added an option that skips the test with vSphere if the OCP version is greater then 4.5

Reply:
@yitzhak12 Is the  bug 1844521 still reproducible in DC-CP?
I suggest to reconfirm once as this DC is better than previous one.
Also can you please update the bug if you are able to repro the issue.

Reply:
Currently, this PR is for deleting an OCS node and replace it with another existing node. I didn't face any issues with that process and also for deploying a cluster with 6 worker nodes on DC-CP. So in the meantime, I will delete the BZ decorator and if the bug will be reproduced will add it again.